test/runner.sh

```diff
@@ -53,7 +53,8 @@ cd ${EXE_DIR}/sql
 
 # create database and install timescaledb
 ${PSQL} $@ -U $TEST_ROLE_SUPERUSER -d postgres -v ECHO=none -c "CREATE DATABASE \"${TEST_DBNAME}\";"
-${PSQL} $@ -U $TEST_ROLE_SUPERUSER -d ${TEST_DBNAME} -v ECHO=none -c "set client_min_messages=error; CREATE EXTENSION timescaledb;"
+${PSQL} $@ -U $TEST_ROLE_SUPERUSER -d ${TEST_DBNAME} -v ECHO=none -c "SET client_min_messages=error; CREATE EXTENSION timescaledb;"
+${PSQL} $@ -U $TEST_ROLE_SUPERUSER -d ${TEST_DBNAME} -v ECHO=none -c "SELECT _timescaledb_internal.stop_background_workers();" > /dev/null
```

Comment:
Can we just add this to `testsupport.sql` instead, which gets included below? Might even make sense for line above that creates the extension.

Reply:
i decided not to do this in case there were other users of testsupport.sql but since test runner is only user i moved it there